hdk/cl/developer_designs/cl_firesim/verif/scripts/top.vcs.f

```diff
@@ -43,6 +44,9 @@
 +incdir+${HDK_COMMON_DIR}/verif/include
 +incdir+${HDK_SHELL_DESIGN_DIR}/ip/cl_debug_bridge/bd_0/ip/ip_0/hdl/verilog
 +incdir+${HDK_SHELL_DESIGN_DIR}/ip/axi_register_slice_light/hdl
++incdir+${CL_ROOT}/design/axi_crossbar_0
```

Comment:
What is this used for?

Reply:
I copied several things from the aws dram_dma example to make sure that xsim will work. I did not copy each line separately, but I'm fairly sure this is not necessary. I can try removing this an making sure that xsim still builds

Reply:
Sounds good.